### DISCUSSION TITLE
Add support for select2

### DISCUSSION THREAD

**Discussion by xcalabuig** [spatie/laravel-livewire-wizard#43] [Q&A]
When working with select2 a lot of issues are encountered, for example:
- issues when setting dynamic options
- selected values dissapears when updating the component or getting back from the next step

**Comment by ju5t** [comment]
This is beyond the scope of this package. If you find a way to handle these things I'm pretty sure @freekmurze might consider adding best practices to the docs, but actual support for very specific use cases shouldn't be part of the package itself.
